chapters/02-filesystem.Rmd

```diff
@@ -1,16 +1,137 @@
+  * demonstrates and reinforces the importance of directory structure and project organization
+* text: accessing the terminal interface in RStudio
+  * locate tab in console window
+  * explain command prompt, and difference between bash and zsh (latter maybe not necessary given commands executed here?)
+
+## Paths and running commands in the Unix shell and understanding paths
+
+* text: basic commands in the Unix shell
+  * there are many commands that perform many different operations
+  * in this chapter we're focusing on commands used for navigating folders and directories
+* code: `pwd` as first command
+* text: interpret output from `pwd`
+  * differing output from Mac and Windows
+  * define working directory
+
+* code: `ls` to show what is in current (working) directory
+* text: describe output from `ls`
+* code: `ls -a` adding an option to show hidden files
+* text: identify differences between this output and earlier output
+  * `.` prefacing a file name means the file is hidden
+  * `.` and `..` as shortcuts to folders (current and parent file, respectively)
+  * define option/flag as way to modify how the command is executed
+* code: `ls Desktop`: include a directory as an argument to show contents of a different directory
+
+* code: `cd Desktop` to change directory, then `pwd` and `ls` to inspect what has happened
+* text: working directory does not necessarily match the file browser in lower right pane
+* text: changing the working directory can make it easier to reference files and directories 
+
+* text: paths represent a location in your computer, and are comprised of directory names separated by slashes (or backslashes?)
+  * absolute paths
+  * relative paths
+  * `~` as home directory
+
+## Creating a project
+
+* text (image?): one of the challenges of software engineering is keeping our code and associated files organized
+  * we use projects to make these tasks easier
+  * if you are looking at RStudio with no project open, you'll see the working directory listed at the top of the Console as `~/` (or Windows equivalent)
+
+* text (image): create a new project
+  * File -> New Project -> New Directory -> New Project
+  * Directory name: `sassafras`
+  * Create project as subdirectory of: `~/Desktop` (or the equivalent on a Windows machine?); may need to use Browse button
+  * Click "Create Project"
+
+* text: what happens after creating a project?
+  * screen refreshes and path to working directory changes (ends in name of project you just created)
+* code: `pwd` and `ls -a` in Terminal to confirm working directory and show files in project
+* text: interpreting output:
+  * if creating a new project, creates a folder in the specified location that includes hidden files containing information about your project
+  * RStudio references that information to make it easier for us to focus on writing code (includes working directory and our preferences)
+
+* text: why aren't we creating a project to use in later chapters?
```

Comment:
I think this is worthwhile (both mentioning it and not creating a project to use later). It's good to get warmed up in a low-stakes way.